src/celengine/framebuffer.h

```diff
@@ -40,28 +42,36 @@ class FramebufferObject
         return m_height;
     }
 
-    GLuint colorTexture() const;
-    GLuint depthTexture() const;
+    GLuint colorAttachment() const;
+    GLuint depthAttachment() const;
```

Comment:
I think this is not a proper update. `*Texture()` should return texture id (0 if not attached), for Renderbuffers you should add new methods.